w-books/src/App/components/Header/styles.css

```diff
@@ -0,0 +1,27 @@
+.Header {
+  background-color: #FFFFFF;
+  box-shadow: 0 5px 5px #DDDDDD;
+  color: white;
+  height: 96px;
+}
+
+.Header-logo {
+  height: 42px;
+  width: 182px;
+  float: left;
+  position: relative;
+  top: 23px;
+  left: 34px;
+}
+
+.Header-menu {
+  position: relative;
+  float: right;
+  top: 26px;
+  right: 49px;
+}
+
+.Header-button {
+  margin-left: 15px;
```

Comment:
margin can be unifiqued by
`margin: 0 15px;`

check this
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties